src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -913,4 +913,7 @@ This version used in: {2}</value>
   <data name="Install_version_0" xml:space="preserve">
     <value>Install version '{0}'</value>
   </data>
-</root>
+  <data name="AddNamedArguments" xml:space="preserve">
```

Comment:
@CyrusNajmabadi It seems that 99% of keys are PascalCase, should I still change it?